rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py

```diff
@@ -190,6 +211,34 @@ def test_exec(config, ssh_con):
     if not attrs["key"]:
         raise TestExecError("Acls lost after bucket resharding, test failure.")
 
+    # test bug 2024408
```

Comment:
@hmaheswa , I think we need to test multiple reshards should not cause the bucket stats to fail on the dr site here along with the bucket quota

Reply:
@viduship , added code for testing multiple reshard causes bucket stats failure, please review again